pom.xml

```diff
@@ -1090,6 +1111,58 @@
                 <version>${spring.version}</version>
             </dependency>
 
+            <dependency>
+                <artifactId>spring-expression</artifactId>
+                <groupId>org.springframework</groupId>
+                <version>${spring.version}</version>
+            </dependency>
+
+            <dependency>
+                <artifactId>spring-test</artifactId>
+                <groupId>org.springframework</groupId>
+                <version>${spring.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-test</artifactId>
+                <version>${spring-boot.version}</version>
```

Comment:
Again, this should be scoped to `test`.  Also, it looks like you've moved this from `dspace-server-webapp` but left out all the dependency exclusions that were in that file.  That seems surprising this works without those.